### ISSUE TITLE
I can not install maui with dotnet 7 and VS 2022 17.4.0 preview 3.0

### ISSUE THREAD

**Issue by migueBarrera** [dotnet/sdk#28613]
### I can not install maui with dotnet 7
I installed vs enterprise 2022 preview  17.4.0 Preview 3.0 but i can not build with maui with .net 7. When i create a new project, vs creates it with net 6. 

If I try to open a project with maui-net7 => The platform version is not present for one or more target platforms, even if a platform has been specified: net7.0-android, net7.0-ios 

I have tried to repair vs preview with the same result.

I tried to run "dotnet workload install maui" but I got this response:
The machine has a pending reboot. Installation will continue, but you may need to restart.
Installing Maui.Core.Ref.android.7.0.0.6889-x64.msi ........... Done
Installing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Installing Maui.Sdk.7.0.0.6889-x64.msi .... Done
Installing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Installing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ...... Done
Installing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Microsoft.Android.Sdk.Windows.33.0.0.152-x64.msi .......... Done
Installing Microsoft.Android.Ref.33.33.0.0.152-x64.msi ..... Done
Installing Microsoft.Android.Runtime.33.android-arm.33.0.0.152-x64.msi ..... Done
Installing Microsoft.Android.Runtime.33.android-arm64.33.0.0.152-x64.msi ..... Done
Installing Microsoft.Android.Runtime.33.android-x86.33.0.0.152-x64.msi ..... Done
Installing Microsoft.Android.Runtime.33.android-x64.33.0.0.152.msi ..... Done
Installing Microsoft.Android.Templates.33.0.0.152-x64.msi .... Done
Downloading Microsoft.NETCore.App.Runtime.Mono.android-arm.Msi.x64 (6.0.11)
Workload installation failed. Rolling back installed packs...
Removing Microsoft.Android.Templates.33.0.0.152-x64.msi .... Done
Removing Microsoft.Android.Runtime.33.android-x64.33.0.0.152.msi .... Done
Removing Microsoft.Android.Runtime.33.android-x86.33.0.0.152-x64.msi .... Done
Removing Microsoft.Android.Runtime.33.android-arm64.33.0.0.152-x64.msi .... Done
Removing Microsoft.Android.Runtime.33.android-arm.33.0.0.152-x64.msi .... Done
Removing Microsoft.Android.Ref.33.33.0.0.152-x64.msi .... Done
Removing Microsoft.Android.Sdk.Windows.33.0.0.152-x64.msi ......... Done
Removing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Removing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ..... Done
Removing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Removing Maui.Sdk.7.0.0.6889-x64.msi ..... Done
Removing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.android.7.0.0.6889-x64.msi .... Done
Workload installation failed: One or more errors occurred. (microsoft.netcore.app.runtime.mono.android-arm.msi.x64::6.0.11 is not found in NuGet feeds https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json;https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json".)

If i execute 'dotnet workload install maui', i get the same response.
I think the problem is in my nuget.congif but I can't solve it.

### My Nuget.Config

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <clear />
    <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" protocolVersion="3" />
    <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
    <add key="dotnet7" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json" />
    <add key="skiasharp" value="https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json" />
    <add key="wasdk-internal" value="https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json" />
  </packageSources>
  <activePackageSource>
    <add key="All" value="(Aggregate source)" />
  </activePackageSource>
  <disabledPackageSources>
    <add key="wasdk-internal" value="true" />
  </disabledPackageSources>
</configuration>
```

### Further technical details
- $ dotnet --info
.NET SDK:
 Version:   7.0.100-rc.2.22477.23
 Commit:    0a5360315a

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\7.0.100-rc.2.22477.23\

Host:
  Version:      7.0.0-rc.2.22472.3
  Architecture: x64
  Commit:       550605cc93

.NET SDKs installed:
  7.0.100-rc.2.22477.23 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.10 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.0-rc.2.22476.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.0-rc.2.22472.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.10 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 7.0.0-rc.2.22472.13 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

Other architectures found:
  arm64 [C:\Program Files\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\arm64\InstallLocation]
  x86   [C:\Program Files (x86)\dotnet]
    registered at [HKLM\SOFTWARE\dotnet\Setup\InstalledVersions\x86\InstallLocation]

Environment variables:
  Not set

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download

- I use vs enterprise 2022 preview  17.4.0 Preview 3.0 on Windows 10

**Comment by migueBarrera** [comment]
I installed vs enterprise 2022 preview 17.4.0 Preview 4.0 with the same result

**Comment by tactical-drone** [comment]
Ok so I fixed the issue by installing `.Net8` and doing the install (`dotnet workload install maui) from there. 

`https://github.com/dotnet/installer/blob/main/README.md#installers-and-binaries`

You can side load .net 7 using `global.json` if you need to be on that version
```
{
  "sdk": {    
    "version": "7.0.100",
    "rollForward": "latestMinor",
      "allowPrerelease": true
  }
}
```

**Comment by migueBarrera** [comment]
@tactical-drone I tried your workaround but no work for me.
I execute "dotnet workload install maui" after install net 8 and this is the response:

`Welcome to .NET 8.0!
---------------------
SDK Version: 8.0.100-alpha.1.22514.14

Telemetry
---------
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.

Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry

----------------
Installed an ASP.NET Core HTTPS development certificate.
To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only).
Learn about HTTPS: https://aka.ms/dotnet-https
----------------
Write your first app: https://aka.ms/dotnet-hello-world
Find out what's new: https://aka.ms/dotnet-whats-new
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------

Advertising manifest not updated. Manifest package for microsoft.net.workload.emscripten.net7 doesn't exist.
Advertising manifest not updated. Manifest package for microsoft.net.workload.mono.toolchain.net6 doesn't exist.
Advertising manifest not updated. Manifest package for microsoft.net.workload.emscripten.net6 doesn't exist.
Skipping NuGet package signature verification.
Advertising manifest not updated. Manifest package for microsoft.net.workload.mono.toolchain doesn't exist.
Advertising manifest not updated. Manifest package for microsoft.net.workload.mono.toolchain.net7 doesn't exist.
The machine has a pending reboot. Installation will continue, but you may need to restart.
Installing Maui.Core.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Installing Maui.Sdk.7.0.0.6889-x64.msi .... Done
Installing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Installing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ...... Done
Installing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Downloading Microsoft.Android.Sdk.Windows.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Sdk.Windows.Msi.x64 (32.0.476)
Downloading Microsoft.Android.Ref.33.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Runtime.33.android-arm.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Runtime.33.android-arm64.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Runtime.33.android-x86.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Runtime.33.android-x64.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.Android.Templates.Msi.x64 (33.0.0-rc.2.202)
Downloading Microsoft.NETCore.App.Runtime.Mono.android-arm.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.Mono.android-x64.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.Mono.android-x86.Msi.x64 (6.0.10)
Downloading Microsoft.NET.Runtime.MonoAOTCompiler.Task.Msi.x64 (6.0.10)
Downloading Microsoft.NET.Runtime.MonoTargets.Sdk.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64.Msi.x64 (6.0.10)
Downloading Microsoft.NETCore.App.Runtime.Mono.android-arm.Msi.x64 (8.0.0-alpha.1.22512.8)
Workload installation failed. Rolling back installed packs...
Removing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Removing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ..... Done
Removing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Removing Maui.Sdk.7.0.0.6889-x64.msi ..... Done
Removing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.android.7.0.0.6889-x64.msi .... Done
Downloading microsoft.net.sdk.android.manifest-7.0.100-rc.2.msi.x64 (33.0.0-rc.2.202)
Workload installation failed: One or more errors occurred. (microsoft.netcore.app.runtime.mono.android-arm.msi.x64::8.0.0-alpha.1.22512.8 is not found in NuGet feeds https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json;https://pkgs.dev.azure.com/xamarin/public/_packaging/SkiaSharp/nuget/v3/index.json".)`

**Comment by tactical-drone** [comment]
@migueBarrera 

You need to add .net 8 feeds to your global nuget sources:
```
<configuration>
  <packageSources>
    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
  </packageSources>
</configuration>
```

**Comment by migueBarrera** [comment]
Thank so much for your help. I have added the nuget you mentioned and I get another error. It's frustrating:

`dotnet workload install maui

Skipping NuGet package signature verification.
The machine has a pending reboot. Installation will continue, but you may need to restart.
Installing Maui.Core.Ref.android.7.0.0.6889-x64.msi .......... Done
Installing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Installing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Installing Maui.Sdk.7.0.0.6889-x64.msi .... Done
Installing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Installing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ...... Done
Installing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Installing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Installing Microsoft.Mono.android-arm.8.0.0-alpha.1.22518.11-x64.msi ...... Done
Installing Microsoft.Mono.android-arm64.8.0.0-alpha.1.22518.11-x64.msi ...... Done
Installing Microsoft.Mono.android-x64.8.0.0-alpha.1.22518.11.msi ...... Done
Installing Microsoft.Mono.android-x86.8.0.0-alpha.1.22518.11-x64.msi ...... Done
Installing Microsoft.MonoAOTCompiler.Task.8.0.0-alpha.1.22518.11-x64.msi .... Done
Installing Microsoft.MonoTargets.Sdk.8.0.0-alpha.1.22518.11-x64.msi .... Done
Installing Microsoft.AOT.win-x64.Cross.android-x86.8.0.0-alpha.1.22518.11.msi ..... Done
Installing Microsoft.AOT.win-x64.Cross.android-x64.8.0.0-alpha.1.22518.11.msi ..... Done
Installing Microsoft.AOT.win-x64.Cross.android-arm.8.0.0-alpha.1.22518.11.msi ..... Done
Installing Microsoft.AOT.win-x64.Cross.android-arm64.8.0.0-alpha.1.22518.11.msi .... Failed
Workload installation failed. Rolling back installed packs...
Removing Microsoft.AOT.win-x64.Cross.android-arm.8.0.0-alpha.1.22518.11.msi .... Done
Removing Microsoft.AOT.win-x64.Cross.android-x64.8.0.0-alpha.1.22518.11.msi .... Done
Removing Microsoft.AOT.win-x64.Cross.android-x86.8.0.0-alpha.1.22518.11.msi .... Done
Removing Microsoft.MonoTargets.Sdk.8.0.0-alpha.1.22518.11-x64.msi .... Done
Removing Microsoft.MonoAOTCompiler.Task.8.0.0-alpha.1.22518.11-x64.msi ..... Done
Removing Microsoft.Mono.android-x86.8.0.0-alpha.1.22518.11-x64.msi .......... Done
Removing Microsoft.Mono.android-x64.8.0.0-alpha.1.22518.11.msi .......... Done
Removing Microsoft.Mono.android-arm64.8.0.0-alpha.1.22518.11-x64.msi .......... Done
Removing Microsoft.Mono.android-arm.8.0.0-alpha.1.22518.11-x64.msi .......... Done
Removing Maui.Essentials.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.any.7.0.0.6889-x64.msi .... Done
Removing Maui.Templates.net7.7.0.0.6889-x64.msi .... Done
Removing Maui.Resizetizer.Sdk.7.0.0.6889-x64.msi ..... Done
Removing Maui.Graphics.7.0.0.6889-x64.msi .... Done
Removing Maui.Sdk.7.0.0.6889-x64.msi ..... Done
Removing AspNetCore.WebView.Maui.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Essentials.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Controls.Ref.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Runtime.android.7.0.0.6889-x64.msi .... Done
Removing Maui.Core.Ref.android.7.0.0.6889-x64.msi .... Done
Workload installation failed: Failed to install Microsoft.AOT.win-x64.Cross.android-arm64.8.0.0-alpha.1.22518.11.msi. Error: 0x00000661. This installation package is not supported by this processor type. Contact your product vendor.
`

**Comment by tactical-drone** [comment]
@migueBarrera I have updates. 

After doing a `dotnet workload install maui` my build broke again. After lots of level pulling what worked was to insert these into the `csproj` file

```
<ItemGroup>
		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0-rtm.22511.4"/>
		<PackageReference Include="Microsoft.Extensions.Logging" Version="7.0.0-rtm.22511.4"/>
		<PackageReference Include="Microsoft.Extensions.Configuration" Version="7.0.0-rtm.22511.4"/>
		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0-rtm.22511.4"/>
</ItemGroup>
```

This seems to fix things for me. 

The entire process seems still a bit wonky. I suggest pulling random levers until something works.

**Comment by tactical-drone** [comment]
@migueBarrera use `https://github.com/Redth/dotnet-maui-check`

**Comment by alfkonee** [comment]
This issues still seems to be persisting and  it a very bad experience for new commers to MAUI so from my investigations here's what I've found
for **.NET 7.0.101 SDK**

For a MAUI workload install using only the public sources I found that `microsoft.android.sdk.windows::33.0.22` and it's related deps have not been promote to the public space although this should be 
```
dotnet workload install maui --source  https://api.nuget.org/v3/index.json

Installing pack Microsoft.Maui.Core.Ref.android version 7.0.52...
Pack Microsoft.Maui.Core.Ref.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Core.Ref.android version 7.0.52...
Installing pack Microsoft.Maui.Core.Runtime.android version 7.0.52...
Pack Microsoft.Maui.Core.Runtime.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.android version 7.0.52...
Installing pack Microsoft.Maui.Controls.Ref.android version 7.0.52...
Pack Microsoft.Maui.Controls.Ref.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.android version 7.0.52...
Installing pack Microsoft.Maui.Controls.Runtime.android version 7.0.52...
Pack Microsoft.Maui.Controls.Runtime.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.android version 7.0.52...
Installing pack Microsoft.Maui.Essentials.Ref.android version 7.0.52...
Pack Microsoft.Maui.Essentials.Ref.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.android version 7.0.52...
Installing pack Microsoft.Maui.Essentials.Runtime.android version 7.0.52...
Pack Microsoft.Maui.Essentials.Runtime.android version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.android version 7.0.52...
Installing pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.52...
Pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.AspNetCore.Components.WebView.Maui version 7.0.52...
Installing pack Microsoft.Maui.Sdk version 7.0.52...
Pack Microsoft.Maui.Sdk version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Sdk.net7 version 7.0.52...
Installing pack Microsoft.Maui.Sdk version 6.0.548...
Pack Microsoft.Maui.Sdk version 6.0.548 is already installed.
Writing workload pack installation record for Microsoft.Maui.Sdk.net6 version 6.0.548...
Installing pack Microsoft.Maui.Graphics version 7.0.52...
Pack Microsoft.Maui.Graphics version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Graphics version 7.0.52...
Installing pack Microsoft.Maui.Resizetizer.Sdk version 7.0.52...
Pack Microsoft.Maui.Resizetizer.Sdk version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Resizetizer.Sdk version 7.0.52...
Installing pack Microsoft.Maui.Templates.net7 version 7.0.52...
Pack Microsoft.Maui.Templates.net7 version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Templates.net7 version 7.0.52...
Installing pack Microsoft.Maui.Templates.net6 version 6.0.548...
Pack Microsoft.Maui.Templates.net6 version 6.0.548 is already installed.
Writing workload pack installation record for Microsoft.Maui.Templates.net6 version 6.0.548...
Installing pack Microsoft.Maui.Core.Ref.any version 7.0.52...
Pack Microsoft.Maui.Core.Ref.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Core.Ref.any version 7.0.52...
Installing pack Microsoft.Maui.Core.Runtime.any version 7.0.52...
Pack Microsoft.Maui.Core.Runtime.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.any version 7.0.52...
Installing pack Microsoft.Maui.Controls.Ref.any version 7.0.52...
Pack Microsoft.Maui.Controls.Ref.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.any version 7.0.52...
Installing pack Microsoft.Maui.Controls.Runtime.any version 7.0.52...
Pack Microsoft.Maui.Controls.Runtime.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.any version 7.0.52...
Installing pack Microsoft.Maui.Essentials.Ref.any version 7.0.52...
Pack Microsoft.Maui.Essentials.Ref.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.any version 7.0.52...
Installing pack Microsoft.Maui.Essentials.Runtime.any version 7.0.52...
Pack Microsoft.Maui.Essentials.Runtime.any version 7.0.52 is already installed.
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.any version 7.0.52...
Installing pack Microsoft.Android.Sdk.Windows version 33.0.22...
Workload installation failed. Rolling back installed packs...
Rolling back pack Microsoft.Android.Sdk.Windows installation...
Workload installation failed: microsoft.android.sdk.windows::33.0.22 is not found in NuGet feeds https://api.nuget.org/v3/index.json".
```
Adding the >NET 8 Engineering Source  Yields
```
 dotnet workload install maui --source  https://api.nuget.org/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json

Installing workload manifest microsoft.net.sdk.maui version 7.0.54…
Installing pack Microsoft.Maui.Core.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Installing pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Writing workload pack installation record for Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Sdk.net7 version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Sdk.net6 version 6.0.550...
Installing pack Microsoft.Maui.Graphics version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Graphics version 7.0.54...
Installing pack Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Installing pack Microsoft.Maui.Templates.net7 version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Templates.net7 version 7.0.54...
Installing pack Microsoft.Maui.Templates.net6 version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Templates.net6 version 6.0.550...
Installing pack Microsoft.Maui.Core.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Installing pack Microsoft.Android.Sdk.Windows version 33.0.22...
Workload installation failed. Rolling back installed packs...
Rolling back pack Microsoft.Android.Sdk.Windows installation...
Rolling back pack Microsoft.Maui.Essentials.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Controls.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Core.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Core.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Templates.net6 installation...
Uninstalling workload pack Microsoft.Maui.Templates.net6 version 6.0.550…
Rolling back pack Microsoft.Maui.Templates.net7 installation...
Uninstalling workload pack Microsoft.Maui.Templates.net7 version 7.0.54…
Rolling back pack Microsoft.Maui.Resizetizer.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Resizetizer.Sdk version 7.0.54…
Rolling back pack Microsoft.Maui.Graphics installation...
Uninstalling workload pack Microsoft.Maui.Graphics version 7.0.54…
Rolling back pack Microsoft.Maui.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Sdk.net6 version 6.0.550…
Rolling back pack Microsoft.Maui.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Sdk.net7 version 7.0.54…
Rolling back pack Microsoft.AspNetCore.Components.WebView.Maui installation...
Uninstalling workload pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.android version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Controls.Ref.android version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Core.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Core.Ref.android version 7.0.54…
Workload installation failed: microsoft.android.sdk.windows::33.0.22 is not found in NuGet feeds https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json".
```

Then adding  .NET 7 engineering source yields

```
dotnet workload install maui --source  https://api.nuget.org/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json

Installing workload manifest microsoft.net.sdk.android version 33.0.26…
Installing workload manifest microsoft.net.sdk.ios version 16.2.1004…
Installing workload manifest microsoft.net.sdk.maccatalyst version 16.2.1004…
Installing workload manifest microsoft.net.sdk.macos version 13.1.1004…
Installing workload manifest microsoft.net.sdk.maui version 7.0.54…
Installing workload manifest microsoft.net.sdk.tvos version 16.1.1501…
Installing pack Microsoft.Maui.Core.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Installing pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Writing workload pack installation record for Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Sdk.net7 version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Sdk.net6 version 6.0.550...
Installing pack Microsoft.Maui.Graphics version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Graphics version 7.0.54...
Installing pack Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Installing pack Microsoft.Maui.Templates.net7 version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Templates.net7 version 7.0.54...
Installing pack Microsoft.Maui.Templates.net6 version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Templates.net6 version 6.0.550...
Installing pack Microsoft.Maui.Core.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Installing pack Microsoft.Android.Sdk.Windows version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Sdk.net7 version 33.0.26...
Installing pack Microsoft.Android.Sdk.Windows version 32.0.485...
Workload installation failed. Rolling back installed packs...
Rolling back pack Microsoft.Android.Sdk.Windows installation...
Rolling back pack Microsoft.Android.Sdk.Windows installation...
Uninstalling workload pack Microsoft.Android.Sdk.net7 version 33.0.26…
Rolling back pack Microsoft.Maui.Essentials.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Controls.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Runtime.any installation...
Uninstalling workload pack Microsoft.Maui.Core.Runtime.any version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Ref.any installation...
Uninstalling workload pack Microsoft.Maui.Core.Ref.any version 7.0.54…
Rolling back pack Microsoft.Maui.Templates.net6 installation...
Uninstalling workload pack Microsoft.Maui.Templates.net6 version 6.0.550…
Rolling back pack Microsoft.Maui.Templates.net7 installation...
Uninstalling workload pack Microsoft.Maui.Templates.net7 version 7.0.54…
Rolling back pack Microsoft.Maui.Resizetizer.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Resizetizer.Sdk version 7.0.54…
Rolling back pack Microsoft.Maui.Graphics installation...
Uninstalling workload pack Microsoft.Maui.Graphics version 7.0.54…
Rolling back pack Microsoft.Maui.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Sdk.net6 version 6.0.550…
Rolling back pack Microsoft.Maui.Sdk installation...
Uninstalling workload pack Microsoft.Maui.Sdk.net7 version 7.0.54…
Rolling back pack Microsoft.AspNetCore.Components.WebView.Maui installation...
Uninstalling workload pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Essentials.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.android version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Controls.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Controls.Ref.android version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Runtime.android installation...
Uninstalling workload pack Microsoft.Maui.Core.Runtime.android version 7.0.54…
Rolling back pack Microsoft.Maui.Core.Ref.android installation...
Uninstalling workload pack Microsoft.Maui.Core.Ref.android version 7.0.54…
Workload installation failed: microsoft.android.sdk.windows::32.0.485 is not found in NuGet feeds https://api.nuget.org/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json;https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json".
``` 

It's only after Added the .NET 6 Engineering source do I get a success 
```
dotnet workload install maui --source  https://api.nuget.org/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json

Installing workload manifest microsoft.net.sdk.android version 33.0.26…
Installing workload manifest microsoft.net.sdk.ios version 16.2.1004…
Installing workload manifest microsoft.net.sdk.maccatalyst version 16.2.1004…
Installing workload manifest microsoft.net.sdk.macos version 13.1.1004…
Installing workload manifest microsoft.net.sdk.maui version 7.0.54…
Installing workload manifest microsoft.net.sdk.tvos version 16.1.1501…
Installing pack Microsoft.Maui.Core.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.android version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.android version 7.0.54...
Installing pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Writing workload pack installation record for Microsoft.AspNetCore.Components.WebView.Maui version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Sdk.net7 version 7.0.54...
Installing pack Microsoft.Maui.Sdk version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Sdk.net6 version 6.0.550...
Installing pack Microsoft.Maui.Graphics version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Graphics version 7.0.54...
Installing pack Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Resizetizer.Sdk version 7.0.54...
Installing pack Microsoft.Maui.Templates.net7 version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Templates.net7 version 7.0.54...
Installing pack Microsoft.Maui.Templates.net6 version 6.0.550...
Writing workload pack installation record for Microsoft.Maui.Templates.net6 version 6.0.550...
Installing pack Microsoft.Maui.Core.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.any version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.any version 7.0.54...
Installing pack Microsoft.Android.Sdk.Windows version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Sdk.net7 version 33.0.26...
Installing pack Microsoft.Android.Sdk.Windows version 32.0.485...
Writing workload pack installation record for Microsoft.Android.Sdk.net6 version 32.0.485...
Installing pack Microsoft.Android.Ref.33 version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Ref.33 version 33.0.26...
Installing pack Microsoft.Android.Runtime.33.android-arm version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Runtime.33.android-arm version 33.0.26...
Installing pack Microsoft.Android.Runtime.33.android-arm64 version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Runtime.33.android-arm64 version 33.0.26...
Installing pack Microsoft.Android.Runtime.33.android-x86 version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Runtime.33.android-x86 version 33.0.26...
Installing pack Microsoft.Android.Runtime.33.android-x64 version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Runtime.33.android-x64 version 33.0.26...
Installing pack Microsoft.Android.Templates version 33.0.26...
Writing workload pack installation record for Microsoft.Android.Templates version 33.0.26...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.android-arm version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.android-arm64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-x64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.android-x64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.android-x64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-x86 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.android-x86 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.android-x86 version 6.0.12...
Installing pack Microsoft.NET.Runtime.MonoAOTCompiler.Task version 6.0.12...
Pack Microsoft.NET.Runtime.MonoAOTCompiler.Task version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NET.Runtime.MonoAOTCompiler.Task.net6 version 6.0.12...
Installing pack Microsoft.NET.Runtime.MonoTargets.Sdk version 6.0.12...
Pack Microsoft.NET.Runtime.MonoTargets.Sdk version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NET.Runtime.MonoTargets.Sdk.net6 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net6.android-x86 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net6.android-x64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net6.android-arm version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net6.android-arm64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.android-arm version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.android-arm version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-arm64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.android-arm64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.android-arm64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-x64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.android-x64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.android-x64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.android-x86 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.android-x86 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.android-x86 version 7.0.1...
Installing pack Microsoft.NET.Runtime.MonoAOTCompiler.Task version 7.0.1...
Pack Microsoft.NET.Runtime.MonoAOTCompiler.Task version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NET.Runtime.MonoAOTCompiler.Task.net7 version 7.0.1...
Installing pack Microsoft.NET.Runtime.MonoTargets.Sdk version 7.0.1...
Pack Microsoft.NET.Runtime.MonoTargets.Sdk version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NET.Runtime.MonoTargets.Sdk.net7 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x86 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net7.android-x86 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-x64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net7.android-x64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net7.android-arm version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.AOT.win-x64.Cross.android-arm64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.AOT.Cross.net7.android-arm64 version 7.0.1...
Installing pack Microsoft.Maui.Core.Ref.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.ios version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.ios version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.ios version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.ios version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.ios version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.ios version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.ios version 7.0.54...
Installing pack Microsoft.iOS.Sdk version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Sdk.net7 version 16.2.1004...
Installing pack Microsoft.iOS.Sdk version 16.2.3...
Writing workload pack installation record for Microsoft.iOS.Sdk.net6 version 16.2.3...
Installing pack Microsoft.iOS.Windows.Sdk version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Windows.Sdk.Aliased.net7 version 16.2.1004...
Installing pack Microsoft.iOS.Windows.Sdk version 16.2.3...
Writing workload pack installation record for Microsoft.iOS.Windows.Sdk.Aliased.net6 version 16.2.3...
Installing pack Microsoft.iOS.Ref version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Ref version 16.2.1004...
Installing pack Microsoft.iOS.Runtime.ios-arm version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Runtime.ios-arm version 16.2.1004...
Installing pack Microsoft.iOS.Runtime.ios-arm64 version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Runtime.ios-arm64 version 16.2.1004...
Installing pack Microsoft.iOS.Runtime.iossimulator-x86 version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Runtime.iossimulator-x86 version 16.2.1004...
Installing pack Microsoft.iOS.Runtime.iossimulator-x64 version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Runtime.iossimulator-x64 version 16.2.1004...
Installing pack Microsoft.iOS.Runtime.iossimulator-arm64 version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Runtime.iossimulator-arm64 version 16.2.1004...
Installing pack Microsoft.iOS.Templates version 16.2.1004...
Writing workload pack installation record for Microsoft.iOS.Templates.net7 version 16.2.1004...
Installing pack Microsoft.NETCore.App.Runtime.Mono.ios-arm version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.ios-arm version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.ios-arm version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.ios-arm64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.ios-arm64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.ios-arm64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.iossimulator-arm64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.iossimulator-x64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.iossimulator-x86 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.ios-arm version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.ios-arm version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.ios-arm version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.ios-arm64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.ios-arm64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.ios-arm64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-arm64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.iossimulator-arm64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.iossimulator-x64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.iossimulator-x86 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.iossimulator-x86 version 6.0.12...
Installing pack Microsoft.Maui.Core.Ref.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.tizen version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.tizen version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.tizen version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.tizen version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.tizen version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.tizen version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.tizen version 7.0.54...
Installing pack Microsoft.Maui.Core.Ref.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.maccatalyst version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.maccatalyst version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.maccatalyst version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.maccatalyst version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.maccatalyst version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.maccatalyst version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.maccatalyst version 7.0.54...
Installing pack Microsoft.MacCatalyst.Sdk version 16.2.1004...
Writing workload pack installation record for Microsoft.MacCatalyst.Sdk.net7 version 16.2.1004...
Installing pack Microsoft.MacCatalyst.Sdk version 16.2.3...
Writing workload pack installation record for Microsoft.MacCatalyst.Sdk.net6 version 16.2.3...
Installing pack Microsoft.MacCatalyst.Ref version 16.2.1004...
Writing workload pack installation record for Microsoft.MacCatalyst.Ref version 16.2.1004...
Installing pack Microsoft.MacCatalyst.Runtime.maccatalyst-x64 version 16.2.1004...
Writing workload pack installation record for Microsoft.MacCatalyst.Runtime.maccatalyst-x64 version 16.2.1004...
Installing pack Microsoft.MacCatalyst.Runtime.maccatalyst-arm64 version 16.2.1004...
Writing workload pack installation record for Microsoft.MacCatalyst.Runtime.maccatalyst-arm64 version 16.2.1004...
Installing pack Microsoft.MacCatalyst.Templates version 16.2.1004...
Writing workload pack installation record for Microsoft.MacCatalyst.Templates.net7 version 16.2.1004...
Installing pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.maccatalyst-arm64 version 7.0.1...
Installing pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64 version 7.0.1...
Pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64 version 7.0.1 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net7.maccatalyst-x64 version 7.0.1...Installing pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-arm64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.maccatalyst-arm64 version 6.0.12...
Installing pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64 version 6.0.12...
Pack Microsoft.NETCore.App.Runtime.Mono.maccatalyst-x64 version 6.0.12 is already installed.
Writing workload pack installation record for Microsoft.NETCore.App.Runtime.Mono.net6.maccatalyst-x64 version 6.0.12...
Installing pack Microsoft.Maui.Core.Ref.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Ref.win version 7.0.54...
Installing pack Microsoft.Maui.Core.Runtime.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Core.Runtime.win version 7.0.54...
Installing pack Microsoft.Maui.Controls.Ref.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Ref.win version 7.0.54...
Installing pack Microsoft.Maui.Controls.Runtime.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Controls.Runtime.win version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Ref.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Ref.win version 7.0.54...
Installing pack Microsoft.Maui.Essentials.Runtime.win version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Essentials.Runtime.win version 7.0.54...
Installing pack Microsoft.Maui.Graphics.Win2D.WinUI.Desktop version 7.0.54...
Writing workload pack installation record for Microsoft.Maui.Graphics.Win2D.WinUI.Desktop version 7.0.54...
Garbage collecting for SDK feature band(s) 7.0.100...
Uninstalling workload pack Microsoft.AspNetCore.Components.WebView.Maui version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Ref.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Ref.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Ref.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Controls.Runtime.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Ref.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Ref.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Ref.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Runtime.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Runtime.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Core.Runtime.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Ref.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.any version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.ios version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Essentials.Runtime.win version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Graphics version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Graphics.Win2D.WinUI.Desktop version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Resizetizer.Sdk version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Sdk.net6 version 6.0.548…
Uninstalling workload pack Microsoft.Maui.Sdk.net7 version 7.0.52…
Uninstalling workload pack Microsoft.Maui.Templates.net6 version 6.0.548…
Uninstalling workload pack Microsoft.Maui.Templates.net7 version 7.0.52…

Successfully installed workload(s) maui maui-windows.
```

I think the team should really take a look at the packages during release and promote the necessary Package versions and Deps  to their appropriate levels.
That should fix all these issues 

**Ideally a user installing from the GA source should just work in my opinion**

**Comment by tactical-drone** [comment]
@alfkonee Good work. Also try doing exactly what you have been doing but from dotnet8 cli, then switch back to dotnet7 by jacking it down with that `global.json` trick:
```
{
  "sdk": {    
    "version": "7.0.200-preview.22551.17",
    "rollForward": "latestMinor",
      "allowPrerelease": true
  }
}
```

**Comment by marcpopMSFT** [comment]
Old issue triage: .net 7 and old VS so closing. Please file a new issue if you're still having problems.
